From 1792aba7b865b5e083019d04f9cccc48757fe321 Mon Sep 17 00:00:00 2001
From: TraHT <traht@fsoft.com.vn>
Date: Wed, 27 Jun 2018 16:40:20 +0700
Subject: Handle apply
 0001-Add-waylandsink-as-default-playing-in-gstreamer-play.patch to fix hunk

Signed-off-by: TraHT <traht@fsoft.com.vn>
---
 ext/wayland/gstwaylandsink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/wayland/gstwaylandsink.c b/ext/wayland/gstwaylandsink.c
index 46a710f..78bd37a 100644
--- a/ext/wayland/gstwaylandsink.c
+++ b/ext/wayland/gstwaylandsink.c
@@ -949,7 +949,7 @@ plugin_init (GstPlugin * plugin)
 
   gst_wl_shm_allocator_register ();
 
-  return gst_element_register (plugin, "waylandsink", GST_RANK_MARGINAL,
+  return gst_element_register (plugin, "waylandsink", GST_RANK_PRIMARY,
       GST_TYPE_WAYLAND_SINK);
 }
 
-- 
2.7.4

